### ISSUE TITLE
[Accessibility][Keyboard][bot name-Test in Web Chat]: Focus is not visible after activating the "New Messages" button.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-WebChat#2608]
**Re-Purposing this bug to mean:**
Fix the tab order of the Test in Web Chat Pane. It is still an accessibility issue just more global then what the OP tester wrote.

**User Impact:**
Users who depend  on keyboard would face feel difficulty if focus is not visible after activating the 'New messages' button.

**Test Environment:**
OS: Version: 1903(OS Build 18956.1000) 
URL: [https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/8d5ceb22-44c7-4367-bfc6-07378096e055/resourceGroups/sddsddd/providers/Microsoft.BotService/botServices/avinash/test](url)
Browser: Edge

**Pre requisite:**
create new web app bot.

**Repro Steps:**
1.open [https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/8d5ceb22-44c7-4367-bfc6-07378096e055/resourceGroups/sddsddd/providers/Microsoft.BotService/botServices/avinash/test](url) in Edge.
2.  Navigate to already created web app bot.
3.  Navigate to 'Test in Web Chat' in bot management and activate it.
4.  Navigate to new message button and activate it.
5.  Observe the issue.

**Actual Result:**
Keyboard focus is not visible after activating the "New Messages" button. After Pressing Enter on New Messages button focus is lost and then on pressing TAB twice focus comes on webchat text field.

**Expected Result:**
Keyboard focus should be visible on the webchat text field after activating the  "New Messages" button.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D&cid=b975c0b5-5927-4362-9028-4428d83fe83b](url)

**Attachments:**
[MAS2.4.3-web app bot-test in web chat.zip](https://github.com/microsoft/BotFramework-WebChat/files/3858629/MAS2.4.3-web.app.bot-test.in.web.chat.zip)
